metadata_validation_conversion/api/views.py

```diff
@@ -30,6 +34,19 @@
                    'ontologies_test', 'summary_ontologies_test'
                    ]
 
+GLOBAL_ALLOWED_INDICES = ['protocol_files_backup', 'compare-index-protocol_files', 'summary_file',
+                          'bovreg_dataset', 'experiment', 'bovreg_experiment', 'ontologies_test',
+                          'compare-index-protocol_analysis', 'protocol_files_test', 'specimen',
+                          'summary_dataset', 'submission_portal_status', 'compare-index-protocol_samples',
+                          'bovreg_analysis', 'protocol_files', 'protocol_analysis','compare-index-analysis',
+                          'ontologies', 'dataset', 'protocol_analysis_test', 'summary_ontologies_test',
+                          'summary_specimen', 'file', 'compare-index-dataset', 'submissions_test',
+                          'submissions', 'portal_status_test', 'compare-index-organism', 'bovreg_study',
+                          'trackhubs', 'summary_ontologies', 'analysis', 'compare-index-file', 'bovreg_file',
+                          'organism', 'summary_organism', 'article', 'compare-index-article', 'protocol_samples_test',
+                          'bovreg_specimen', 'protocol_samples', 'bovreg_organism', 'faang_subscriptions',
+                          'compare-index-specimen', 'ensembl_annotation']
+
```

Comment:
we don't need to have a search over all these indices and on the FE we will need to link search results to the actual tables, so I would consider to restrict search indices to these one: organism, specimen, dataset, file, analyses, protocols, publications

Reply:
views.py needs to be updated with the latest changes from the master branch before it can be merged. This file is outdated.